File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

    int val;
    do{
    val = get_int("Enter value 1-8 \n");


    }while(val<1 || val>8);


    for(int i=0; i<val; i++){
            int k = i;
            while(k<val-1){
                printf(" ");
                k++;
            }
        for(int j=0;j<i+1;j++)
        {

            printf("#");
        }

        printf("\n");
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int


while True:
    n= get_int("Height: ")

    if n<9 and n>0:
        for i in range(n-1):
            print(" " *(n-(i+2)), "#" *(i+1) , "", "#"*(i+1))
        print("#" * n,"","#"*n)
        break